rasa_sdk/events.py

```diff
@@ -86,12 +88,16 @@ def ReminderScheduled(
 
 # noinspection PyPep8Naming
 def ReminderCancelled(
-    action_name: Text, name: Optional[Text] = None, timestamp: Optional[float] = None
+    name: Optional[Text] = None,
+    intent_name: Optional[Text] = None,
+    entities: Optional[Union[List[Dict[Text, Any]], Dict[Text, Text]]] = None,
+    timestamp: Optional[float] = None,
 ) -> EventType:
```

Comment:
can we add a warning in case `intent_name` starts with `utter_` or `action_`? That's also sth we could unit test then.

Reply:
Done. I am not sure if it was correct to declare it as `DeprecationWarning`, though.

Reply:
Oh, and doing unit tests now...

Reply:
Should be a `FutureWarning`

Reply:
Ah, ok. I thought it's meant for the developers who develop the bot. Not for the enduser. So why is it `Future`?

Reply:
I've changed it now.